pkg/kube/snapshot/snapshot.go

```diff
@@ -102,6 +102,8 @@ type Snapshotter interface {
 	// WaitOnReadyToUse will block until the Volumesnapshot in namespace 'namespace' with name 'snapshotName'
 	// has status 'ReadyToUse' or 'ctx.Done()' is signalled.
 	WaitOnReadyToUse(ctx context.Context, snapshotName, namespace string) error
+	// List will list the volumesnapshots in a namespace that match search
```

Comment:
Should the comment say  "List will.....that match with the given labels"  . What is the behavior when no labels are provided ? Will it return no snapshots or all snapshots ?

Reply:
All snapshots in the namespace.